inbm/dispatcher-agent/tests/unit/source/test_source_command.py

```diff
@@ -164,9 +164,9 @@ def test_do_source_command_remove(
             "dispatcher.source.source_command.create_application_source_manager",
             OsType.Ubuntu,
             ApplicationAddSourceParameters(
-                source_list_file_name="repofilename",
                 gpg_key_name="keyname",
                 gpg_key_uri="gpguri",
+                source_list_file_name="repofilename",
                 sources=["sourceA", "sourceB"],
```

Comment:
**Suggestion:** Ensure consistent ordering of parameters when creating instances of ApplicationAddSourceParameters for readability and maintainability. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            ApplicationAddSourceParameters(
                source_list_file_name="repofilename",
                gpg_key_name="keyname",
                gpg_key_uri="gpguri",
                sources=["sourceA", "sourceB"],
            ),
```